src/test/java/guitests/FilterTextFieldTest.java

```diff
@@ -121,6 +127,21 @@ public void cancelTest() {
         awaitCondition(() -> toggle.get() % 2 == 1);
     }
 
+    @Test
+    public void testHelp() {
```

Comment:
What method or feature are we testing? The method name should be exactly that (i.e. omit `test` part as it is already captured in `@Test`)